### ISSUE TITLE
Intermittent WriteCodeFramentEx failure on dev machines

### ISSUE THREAD

**Issue by agocke** [dotnet/roslyn#9856]
The failure looks like:

```
The "Roslyn.MSBuild.Util.WriteCodeFragmentEx" task could not be loaded from the assembly
C:\Users\angocke\.nuget\packages\Roslyn.Build.Util\0.9.2-portable\lib\dotnet\Roslyn.MSBuild.Util.dll.
Could not load file or assembly 'Microsoft.Build.Utilities.Core, Version=14.1.0.0, Culture=neutral,
PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file
specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its
dependencies are available, and that the task contains a public class that implements
Microsoft.Build.Framework.ITask.
```

**Comment by agocke** [comment]
I'm still not quite sure what's going on here, but I think referencing the framework should cause this reference to unify with the framework on Desktop?

**Comment by davkean** [comment]
This is fixed, yes?

**Comment by agocke** [comment]
It should be, yes

**Comment by agocke** [comment]
Fixed by #9859
